src/components/colorPicker/index.tsx

```diff
@@ -100,8 +103,9 @@ class ColorPicker extends PureComponent<Props> {
           animatedIndex={this.animatedIndex}
           onValueChange={this.onValueChange}
           testID={`${testID}-palette`}
+          backgroundColor={backgroundColor}
         />
-        <View style={styles.buttonContainer}>
+        <View style={[styles.buttonContainer, {backgroundColor}]}>
```

Comment:
BackgroundColor should not be passed here but to the component View wrapper (line 97)